Academia-750/academia750-frontend-vue
# DBT-802 Add Materials to a lesson

## Context


We are adding materials to lesson



## Test



- Login as admin
- got to manage lesson
- click one lesson and in the top click on lesson
- add a lesson/ delete and check the list of the lesson



## Platform
google chrome



## Proof of Concept
